packages/react-accordion/src/components/AccordionItem/AccordionItem.types.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utilities';
+
+export interface AccordionItemContext {
+  headingId: string;
+  panelId: string;
+  open: boolean;
+  onAccordionHeaderClick(ev: React.MouseEvent<HTMLElement>): void;
```

Comment:
```suggestion
  onHeaderClick(ev: React.MouseEvent<HTMLElement>): void;
```

In v0 we include only a slot name there. Not sure about the approach for converged components.